packages/manager/modules/anthos/src/dashboard/host/remove/routing.js

```diff
@@ -0,0 +1,22 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('anthos.dashboard.host.remove', {
+    url: '/remove',
+    params: {
+      host: null,
+    },
+    views: {
+      modal: {
+        component: 'anthosRemoveHost',
+      },
+    },
+    layout: 'modal',
+    resolve: {
+      breadcrumb: () => null,
+      goBack: /* @ngInject */ (goToHost) => (message, type) =>
+        goToHost(message, type),
+      host: /* @ngInject */ ($transition$) => $transition$.params().host,
+      hostService: /* @ngInject */ (AnthosTenantsService, host, serviceName) =>
+        AnthosTenantsService.getHostService(serviceName, host.id),
```

Comment:
getHostService first gets serviceId and then service based on serviceId. serviceId might be already available. Can you check this and remove additional call to getServiceInfo?

Reply:
The service id is available only in the serviceInfo... Hence, we will have to make both the calls...